hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/kafka/KafkaFacade.java

```diff
@@ -670,8 +183,8 @@ public void updateTopicAcl(Project project, String topicName, Integer aclId, Acl
     //remove previous acl
     em.remove(ta);
 
-    //add the new acls  
-    addAclsToTopic(topicName, project.getId(), aclDto);
+    //add the new acls
+    kafkaController.addAclsToTopic(topicName, project.getId(), aclDto);
```

Comment:
@kai-chi KafkaController should not be referenced by the facade.

Reply:
correct, this will be refactored along with the ACL PR. All controller references will be removed PR by PR

Reply:
Yes but we can't merge code where the transactionless controller is used by the facade. If we need to merge topics and acls together, we'll do that then.

Reply:
I will remove the reference in this PR since it's the only one left.